app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -439,8 +439,4 @@
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_start">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_end">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_widget_switch_compat_margin_end">4dp</dimen>
-
-  <!-- OptionAudioLanguage -->
-  <dimen name="option_audio_language_padding_start">16dp</dimen>
-  <dimen name="option_audio_language_padding_end">16dp</dimen>
```

Comment:
Why are these getting removed? They don't seem to be part of this PR? Maybe merging with latest develop can fix this automatically.

Reply:
done